<launch>
    <param name="robot_description" command="cat $(find my_arm)/models/jaco_robot.urdf" />

    <!-- send fake joint values -->
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="TRUE"/>
    </node>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node name="jacoArmController" pkg="my_arm" type="jacoArmController" />

</launch>
